src/ts/src/alog.ts

```diff
@@ -29,6 +29,7 @@ import { Writable } from 'stream';
 import { AlogCoreSingleton } from './core';
 
 // Pass-Through exports
+export { ChannelLog, useChannel } from './channel_log';
```

Comment:
```suggestion
export { ChannelLog, useChannel } from './channel-log';
```

Not python 😁

Reply:
Should the file name change too? This is _definitely_ a `python` dev wearing a `ts` costume...

Reply:
Renamed both the file and this line